pom.xml

```diff
@@ -168,78 +218,73 @@
     </profiles>
 
     <dependencies>
+        <!-- all scopes-->
```

Comment:
I find _"all scopes"_ a bit confusing, as [each dependency has exactly one `<scope>`](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-scope), and the default scope is `compile` if none is specified.

Maybe rather _"all classpaths"_?